exercises/07-normalizr/backend/server.ts

```diff
@@ -0,0 +1,65 @@
+import express from 'express'
+
+import { User, UserName } from '../src/modules/users/user-types'
+import { computeRegnalNumber, hasSameName, isUserName } from './utils'
+
+const port = 3001
+
+const skills = [
+  {
+    id: 'skill-1',
+    name: 'Change my face',
+  },
+  {
+    id: 'skill-2',
+    name: 'Control dragons',
+  },
+]
+
+const recognizedUsers: { [key: string]: UserName } = {
+  AryaStark: { firstName: 'Arya', lastName: 'Stark' },
+  DaenerysTargaryen: { firstName: 'Daenerys', lastName: 'Targaryen' },
+}
+
+const users: User[] = []
+
+const app = express()
+app.use(express.urlencoded({ extended: true }))
+app.use(express.json())
+
+app.get('/users', (_, res) => res.json(users))
+
+app.post('/users', (req, res) => {
+  const userName = req.body
```

Comment:
If I understand it correctly, you named this variable like this because of the `UserName` interface, right? Honestly, it's a bit confusing to me. If I see `userName`, I think it's a `string`. Maybe it's just my psychological problem :smiley: but maybe, we could use a better name for this variable and the interface.

What do you think?

Reply:
I've just renamed `UserData` to `UserName` after https://github.com/salsita/react-training/pull/59#discussion_r471567361 review comment. I'm having troubles thinking of better interface name for this to be honest. It's a name, just divided into two properties.

Reply:
Hmm, I see. Actually, I like `UserData` more. :smile: The name looks like it describes _an object containing all data required to create a new user_. Right now, `firstName` and `lastName` only - right. Once we would add something more (like `age`), the current naming won't be good. I know we will never extend the tasks but... :smile: Or maybe we can be even more specific like `CreateUserData` or something like that. Or we can keep it as it is. :smiley: 

cc @novellizator